doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -177,7 +170,7 @@ satisfy the following requirements as entry pass:
   * Design, Configuration, Features, SLAs, and Capability documentation
     complete
   * Users stories / Adherence to CNTT Model principles and guidelines
-  * Chosen Reference Architecture Matches a Reference Architecture from the
+  * Chosen Reference Architecture Matches the Architecture from the
```

Comment:
nit: Matches -> matches

Reply:
@deebhatia addressed in https://github.com/cntt-n/CNTT/pull/1494/commits/24a343f19605daec66bbcb1b69936e5c1f02bb44